commands/command_prune.go

```diff
@@ -176,28 +178,36 @@ func prune(fetchPruneConfig lfs.FetchPruneConfig, verifyRemote, dryRun, verbose
 	}
 
 	if len(prunableObjects) == 0 {
-		Print("Nothing to prune")
 		return
 	}
+
+	info := tasklog.NewSimpleTask()
+	logger.Enqueue(info)
 	if dryRun {
-		Print("%d files would be pruned (%v)", len(prunableObjects), humanize.FormatBytes(uint64(totalSize)))
+		info.Logf("prune: %d file(s) would be pruned (%s)", len(prunableObjects), humanize.FormatBytes(uint64(totalSize)))
 		if verbose {
-			Print(verboseOutput.String())
+			for _, item := range verboseOutput {
+				info.Logf("\n * %s", item)
+			}
 		}
+		info.Complete()
 	} else {
-		Print("Pruning %d files, (%v)", len(prunableObjects), humanize.FormatBytes(uint64(totalSize)))
 		if verbose {
-			Print(verboseOutput.String())
+			for _, item := range verboseOutput {
+				info.Logf("\n%s", item)
+			}
 		}
+		info.Complete()
 
 		// Since the above progress will have completed, create and
-		// enqueue a _new_ spinner to track deletion progress.
-		spinner := progress.NewSpinner()
-		logger.Enqueue(spinner)
+		// enqueue a _new_ percentage task to track deletion progress.
+		percentage := tasklog.NewPercentageTask("prune: Deleting objects", uint64(len(prunableObjects)))
+		logger.Enqueue(percentage)
```

Comment:
Since this `percentage` is only used inside `pruneDeleteFiles()`, why not pass `logger` to it instead. Same justification as my note on `pruneTaskDisplayProgress()`.